src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -218,6 +218,7 @@ private static SslClientAuthenticationOptions ConstructSslOptions(HttpConnection
         {
             if (cancellationToken.IsCancellationRequested)
             {
+                if (NetEventSource.IsEnabled) Trace("Unable to complete getting HTTP/1.1 connection due to requested cancellation.");
```

Comment:
strictly speaking, the connection might not really be HTTP/1.1 but rather HTTP/1.0.  I realize that the method is called "GetHttp11ConnectionAsync".  But that is only because it was refactored to differentiate from the HTTP/2.0 connection code.  I would have preferred this method be named "GetHttpConnectionAsync" so that its name was more generally applicable to either HTTP/1.0 or HTTP/1.0.  There is already an "Http2" class related to HTTP/2.0 connections.

Perhaps the string "getting HTTP/1.1 connection" can be changed to "getting HTTP/1.x connection".

Reply:
Make sense, will rename the string.